libarchive/ffi.py

```diff
@@ -127,9 +127,12 @@ def ffi(name, argtypes, restype, errcheck=None):
 ffi('entry_rdevmajor', [c_archive_entry_p], c_uint)
 ffi('entry_rdevminor', [c_archive_entry_p], c_uint)
 
-ffi('entry_set_size', [c_archive_entry_p, c_longlong], c_int)
+ffi('entry_set_size', [c_archive_entry_p, c_longlong], c_longlong)
```

Comment:
While I was at it I changed the argument type of `entry_set_filetype` from `c_int` to `c_uint` since the C signature says `unsigned int`.